src/Psy/Command/EditCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Psy\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class EditCommand extends Command
+{
+    private $temporaryDirectory = '';
```

Comment:
call it `runtimeDir` for consistency with XDG and everywhere else we call it that?